Readme

```diff
@@ -1,4 +1,25 @@
-/usr/sbin/mem -u -s 0x1e10b200 -w 0x4024	# MII_CFG_5 (mii_base + portnum*8) = MII_CFG_EN | MII_CFG_RATE_M125 | MII_CFG_MODE_RGMII
-/usr/sbin/mem -u -s 0x1e10b204 -w 0xc003	# PCDU_5
-/usr/sbin/mem -u -s 0x1e10a40c -w 0x0180	# MAC_CTRL_0 todo :switch settings
-/usr/sbin/mem -u -s 0x1e10b140 -w 0x3205	# PHY_ADDR_5 mdio_base + (0x54 -(portnum*4)) = MDIO_PHY_LINK_UP | MDIO_PHY_SPEED_G1 | MDIO_PHY_FDUP_EN | phyaddr
\ No newline at end of file
+
+RaLink driver port
+
+To use them compile the kernel and busybox with devmem support.
+
+Then activate the driver with the following commands:
+busybox devmem 0x1e10b200 32 0x4024 # MII_CFG_5 (mii_base + portnum*8) = MII_CFG_EN | MII_CFG_RATE_M125 | MII_CFG_MODE_RGMII
+busybox devmem 0x1e10b204 32 0xc003 # PCDU_5
+busybox devmem 0x1e10a40c 32 0x0180 # MAC_CTRL_0 todo :switch settings
+busybox devmem 0x1e10b140 32 0x3205 # PHY_ADDR_5 mdio_base + (0x54 -(portnum*4)) = MDIO_PHY_LINK_UP | MDIO_PHY_SPEED_G1 | MDIO_PHY_FDUP_EN | phyaddr
```

Comment:
Are there some other notes necessary regarding the bridge feature?
Is it really needed to modprobe a kernel bridge module? ( "Warning! Bridge module not init yet. Please modprobe/insmod bridge at first if you want to use bridge.\n");" )
I thought it is integrated within the kernel, because the br-lan is created and used. Every time I load the driver this message appears. What do I missing?